src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -156,60 +163,99 @@ const Packages = ({ defaultArch, ...props }) => {
         };
     });
 
-    // move selected packages
-    const moveSelected = (fromAvailable) => {
-        const sourcePackages = fromAvailable ? packagesAvailable : packagesChosen;
-        const destinationPackages = fromAvailable ? packagesChosen : packagesAvailable;
+    const areFound = (filter, packageList) => {
+        return filter === undefined ? true :
+            packageList.some(pack => pack.name.includes(filter)) ? true : false;
+    };
+
+    const isHidden = (filter, pack) => {
+        const filterActive = filter === undefined ? false : true;
```

Comment:
I don't believe this is necessary. In logic I believe undefined evaluates to false. So you can just check for `filterActive && !pack...`

Reply:
Nice catch, I'm still getting comfortable with JS truthiness. Done.